src/tye_lab_to_nwb/general_interfaces/discriminationtaskeventsdatainterface.py

```diff
@@ -7,14 +7,16 @@
 from neuroconv.utils import FilePathType
 
 
-class NeurotensinEventsInterface(TimeIntervalsInterface):
+class DiscriminationTaskEventsInterface(TimeIntervalsInterface):
```

Comment:
Decided to move this to `general_interfaces/ ` as it is used by two conversions.